pyproject.toml

```diff
@@ -39,7 +39,6 @@ requests="2.26.0"
 pytest="7.4.3"
 # use the latest main, switch to PyPi when released
 opustrainer = {git = "https://github.com/hplt-project/OpusTrainer.git", rev="9133e1525c7ee37f53ea14ee6a180152bf7ea192"}
-pytest-clarity = "^1.0.1"
```

Comment:
I thought this would give us nicer diffs, but in CI you don't get the +-. Locally you get red/green colors, but in CI there is no color output.